modules/ensemble/lib/widget/input/form_textfield.dart

```diff
@@ -156,6 +156,8 @@ abstract class BaseTextInput extends StatefulWidget
       'maxLines': (value) =>
           _controller.maxLines = Utils.optionalInt(value, min: 1),
       'textStyle': (style) => _controller.textStyle = Utils.getTextStyle(style),
+      'autofillHints': (value) =>
```

Comment:
@hemish11 this won't work when someone is setting `autofillHints` in js. 

```js
myField.autofillHints = ['a','b'];
```
look at the implementation of `Utils.getListOfStrings` it's terrible code that returns null if it is not yaml. 

Please ALWAYS test the js property setting too. Your code is hardcoded to work only for yaml